lightkurve/correctors/pldcorrector.py

```diff
@@ -85,52 +88,158 @@ class PLDCorrector(Corrector):
-            included in the noise model. You may also pass an array of indices.
-            This option enables signals of interest (e.g. planet transits)
-            to be excluded from the noise model, which will prevent over-fitting.
-            By default, no cadences will be masked.
-        gp_timescale : float
-            Gaussian Process time scale length term (`tau`) used to define
-            length of fit variability in days.
-        use_gp : boolean
-            Option to turn GP fitting on or off.  You would typically only set
-            this to False to speed up the correction (at the cost of precision),
-            or if you suspect the presence of systematic noise at long timescales.
         pld_order : int
             The order of Pixel Level De-correlation to be performed. First order
             (`n=1`) uses only the pixel fluxes to construct the design matrix.
             Higher order populates the design matrix with columns constructed
             from the products of pixel fluxes.
+        pixel_components : int
+            Number of principal components derived from the background pixel
+            time series to utilize.
+        background_mask : array-like or None
+            A boolean array flagging the background pixels such that `True` means
+            that the pixel will be used to generate the background systematics model.
+            If `None`, all pixels which are fainter than 1-sigma above the median
+            flux will be used.
+        pld_aperture_mask : array-like, 'pipeline', 'all', 'threshold', or None
+            A boolean array describing the aperture such that `True` means
+            that the pixel will be used when selecting the PLD basis vectors.
+            If `None` or `all` are passed in, all pixels will be used.
+            If 'pipeline' is passed, the mask suggested by the official pipeline
+            will be returned.
+            If 'threshold' is passed, all pixels brighter than 3-sigma above
+            the median flux will be used.
+        spline_n_knots : int
+            Number of knots in spline.
+        spline_degree : int
+            Polynomial degree of spline.
         n_pca_terms : int
             Number of terms added to the design matrix from each order of PLD
             when performing Principal Component Analysis for models higher than
             first order. Increasing this value may provide higher precision at
             the expense of computational time.
+        sparse : bool
+            Whether to create `SparseDesignMatrix`.
+
+        Returns
+        -------
+        dm : `.DesignMatrixCollection`
+            `.DesignMatrixCollection` containing pixel, background, and spline
+            components.
+        """
+
+        if background_mask is None:
+            # Default to pixels <1-sigma above the background
+            background_mask = ~self.tpf.create_threshold_mask(3, reference_pixel=None)
```

Comment:
The docstring and comment above this line say that we use threshold=1, but the code uses threshold=3.  Is there a good reason?  I'd say threshold=1 is a better background mask?
```suggestion
            background_mask = ~self.tpf.create_threshold_mask(1, reference_pixel=None)
```

Reply:
Yes, I was testing different values and forgot to change this back. Go ahead and commit this change!